scripts/build.ps1

```diff
@@ -53,7 +53,7 @@ param (
     [string]$Config = "Debug",
 
     [Parameter(Mandatory = $false)]
-    [ValidateSet("x86", "x64", "arm", "arm64")]
+    [ValidateSet("x86", "x64", "arm", "arm64", "x86_uwp", "x64_uwp", "arm_uwp", "arm64_uwp")]
```

Comment:
Might it be better just to add a `$UWP` parameter instead that defaults to `$false`?

Reply:
I can do that. It will be a bit more invasive because of modifications required to the directory setting, but that's ok.

Reply:
Yeah, but I think it will end up being cleaner.

Reply:
Fixed